core/settings/qPrefUnit.cpp

```diff
@@ -94,13 +94,17 @@ QString qPrefUnits::unit_system()
 }
 void qPrefUnits::set_unit_system(const QString& value)
 {
-	short int v = value == QStringLiteral("metric") ? METRIC :
+	set_unit_system(value == QStringLiteral("metric") ?  METRIC :
 					value == QStringLiteral("imperial")? IMPERIAL :
-							PERSONALIZE;
-	if (v == METRIC) {
+							                             PERSONALIZE);
```

Comment:
that looks like odd whitespace - I think this may be Xcode introducing its own ideas of continuation line indentation...

Reply:
removed